DDDEastMidlandsLimited/dddem-web
# Bug: Fix Progressive Web App Errors Highlighted By Lighthouse #216

Issue's link: https://github.com/DDDEastMidlandsLimited/dddem-web/issues/216

Fixes:

**Fast and Reliable:** 
- [X] "The start_url did response, but not via a service worker" : it should be already fixed with Chrome 89, try to update your browser. I'll link you the bug/issue : https://github.com/antfu/vite-plugin-pwa/issues/20

**PWA optimized:** 
- [X] is not configured for a custom splash screen : I provided a manifest icon 512 x 512 png
- [X] manifest doesn't have a maskable icon: I provided a maskable icon 192 x 192 png. Only one is needed

How to test:
Open the site in Chrome
Open developer tools
Go to the Lighthouse tab in the dev tools
Run and see if these 3 issues are fixed, you should see:

1.  "Registers a service worker that controls page and start_url " for the Fast and Reliable' s issue
2. "Configured for a custom splash screen" for the Pwa optimized's first issue
3. "Manifest has a maskable icon" for the Pwa optimized's second issue

Disclaimer: it's normal to have errors like "Does not redirect HTTP traffic to HTTPS" etc.. because we are in local/dev env (localhost) and not in prod env.
